Mathlib/Topology/MetricSpace/Kuratowski.lean

```diff
@@ -14,35 +14,35 @@ import Mathlib.Topology.Sets.Compacts
 /-!
 # The Kuratowski embedding
 
-Any separable metric space can be embedded isometrically in `ℓ^∞(ℝ)`.
+Any separable metric space can be embedded isometrically in `ℓ^∞(ℕ, ℝ)`.
+Any partially defined Lipschitz map into `ℓ^∞` can be extended to the whole space.
+
 -/
 
 
 noncomputable section
 
 set_option linter.uppercaseLean3 false
 
-open Set Metric TopologicalSpace
-
-open scoped ENNReal
+open Set Metric TopologicalSpace NNReal ENNReal
 
-local notation "ℓ_infty_ℝ" => lp (fun n : ℕ => ℝ) ∞
+local notation "ℓ^∞(" ι ") " => lp (fun i : ι => ℝ ) ∞
 
 universe u v w
 
 variable {α : Type u} {β : Type v} {γ : Type w}
 
 namespace KuratowskiEmbedding
 
-/-! ### Any separable metric space can be embedded isometrically in ℓ^∞(ℝ) -/
+/-! ### Any separable metric space can be embedded isometrically in ℓ^∞(ℕ, ℝ) -/
```

Comment:
Should it be this?
```suggestion
/-! ### Any separable metric space can be embedded isometrically in ℓ^∞(ℕ) -/
```

Reply:
It seems harmless to abuse notation a bit in docstrings and write this more informative version with both domain and codomain specified ...